src/modules/map/components/openlayers/utils/addLayerToMap-mixins.js

```diff
@@ -26,8 +26,13 @@ const addLayerToMapMixin = {
     },
     unmounted() {
         if (this.layer && this.isPresentOnMap) {
+            this.layer.getSource()?.clear?.()
```

Comment:
I'm a bit on the fence about this notation.
It's simple and short. But (at least for now) unfamiliar. Especially the `clear?.()` part.

Feel free to tell me to do it the classic way! :smiley:

Reply:
I'm not sure you need the last `?.()` part, `this.layer.getSource()?.clear()` should be good enough (`clear()` will only be called if `getSource()` gives something)

Reply:
The problem is that not all sources have a clear method. So it is possible that `getSource()` returns something but `.clear` is undefined.

I might be thinking too far. But from what I see we have the following options:
* Optional chaining  
  Short but unfamiliar.
* Try/catch with an empty catch  
  Somewhat hacky.
* If-block  
  More lines but easy to read.